content/static/js/custom.cfr.js

```diff
@@ -1,4 +1,7 @@
 /************************************************** NMSU CFR ADDITIONAL SCRIPTS **************************************************/
+var testing = "true";
```

Comment:
[NON-ESSENTIAL]These should probably be booleans instead of strings. They also aren't very descriptive names because I'm not sure what they're for.